### PR TITLE
fix(core): bavigate user to parent flow on shared key generations

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA - [LW-10985](https://input-output.atlassian.net/browse/LW-10985)
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

Describe here, how the new implementation can be tested.
Provide link or briefly describe User Acceptance Criteria/Tests that need to be met

## Screenshots

Attach screenshots here if implementation involves some UI changes

[LW-10985]: https://input-output.atlassian.net/browse/LW-10985?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/shared-wallets/add-shared-wallet/generate-key-flow/GenerateSharedWalletKeyFlow.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const GenerateSharedWalletKeyFlow: VFC<GenerateSharedWalletKeyFlowProps>
         )}
         {state.step === GenerateSharedWalletKeyStep.CopyKey && (
           <CopyKey
-            onClose={() => dispatch({ type: ActionType.CloseFlow })}
+            onClose={navigateToParentFlow}
```
Comment (szymonmaslowski) [inline packages/core/src/shared-wallets/add-shared-wallet/generate-key-flow/GenerateSharedWalletKeyFlow.tsx:45]: It already works this way. `CloseFlow` action gets handled by the machine which calls the navigateToParentFlow [here](https://github.com/input-output-hk/lace/blob/a4890581b8770f1f3ddeee276cd9c6cc092c5df9/packages/core/src/shared-wallets/add-shared-wallet/generate-key-flow/Store/machine.ts#L82).

File: packages/translation/src/lib/translations/shared-wallets/en.json [hunk 2]
```diff
@@ -80,7 +80,7 @@
   "sharedWallets.addSharedWallet.keyGeneration.copyKeys.title": "Copy shared wallet key",
   "sharedWallets.addSharedWallet.keyGeneration.copyKeys.subtitle": "Copy and send this key to someone setting up a new shared wallet. You can review this key at any time.",
   "sharedWallets.addSharedWallet.keyGeneration.copyKeys.backButtonLabel": "Copy key to clipboard",
-  "sharedWallets.addSharedWallet.keyGeneration.copyKeys.nextButtonLabel": "Close",
+  "sharedWallets.addSharedWallet.keyGeneration.copyKeys.nextButtonLabel": "Next",
```
Comment (szymonmaslowski) [inline packages/translation/src/lib/translations/shared-wallets/en.json:83]: According to the designs, this button should have text "Close": https://www.figma.com/design/RPN3V4Et7hMbXkV5ZIF75i/%5Bdeprecated%5D-Lace-Browser---Full-view-UI-Design?node-id=43369-55703&t=xqtJQq6Ys6UsTKPu-0

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]

File: packages/translation/src/lib/translations/shared-wallets/en.json (hunk 2 above)
Comment (vetalcore) [reply packages/translation/src/lib/translations/shared-wallets/en.json:83]: it makes more sense if we name it closer to the action it represents?

Comment (vetalcore) [reply packages/translation/src/lib/translations/shared-wallets/en.json:83]: also designs are something we should question

File: packages/core/src/shared-wallets/add-shared-wallet/generate-key-flow/GenerateSharedWalletKeyFlow.tsx (hunk 1 above)
Comment (vetalcore) [reply packages/core/src/shared-wallets/add-shared-wallet/generate-key-flow/GenerateSharedWalletKeyFlow.tsx:45]: nope

**Comment by pczeglik-iohk** [comment]
# Allure Report
[`allure-report-publisher`](https://github.com/andrcuns/allure-report-publisher) generated test report!



**smokeTests**: ✅ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/smoke/linux/chrome/6326/10041965469/index.html) for [a4890581](https://github.com/input-output-hk/lace/pull/1313/commits/a4890581b8770f1f3ddeee276cd9c6cc092c5df9)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 32     | 0      | 0       | 0     | 32    | ✅     |

File: packages/translation/src/lib/translations/shared-wallets/en.json (hunk 2 above)
Comment (vetalcore) [reply packages/translation/src/lib/translations/shared-wallets/en.json:83]: @danielmain wdyt?

**Pull request closed without merging**